modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -29,48 +34,61 @@ You cannot add a local registry to any version of an existing Runtime Fabric ins
 
-Ensure that you've performed the following tasks: 
+[tabs]
+====
+Synchronize Automatically:: 
 
-. Set up, configure, and test your private Docker image registry.
-. Synchronize to your local registry all the Docker images you need to install Runtime Fabric:
-+
-See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes-2.x.x.adoc[Runtime Fabric release notes] for the required dependencies for your Runtime Fabric version. 
 +
+--
+To synchronize the dependencies images automatically, you ran a bash script that uses `docker`, `curl`,  and `jq` tools.
+
 [NOTE]
-Dependency versions are specific to the Runtime Fabric version.
- 
+Automated registry synchronization is available from Runtime Fabric version 2.6.0 onwards. Previous dependencies can not be retrieved using this script.
 
-. xref:install-self-managed.adoc#step-3-download-the-rtfctl-utility[Download and install the `rtfctl` command line utility] version 0.3.150 or higher.
-. xref:install-self-managed.adoc#step-3-download-the-rtfctl-utility[Create a Runtime Fabric using Runtime Manager], and gather the activation data. 
+Ensure that your xref:access-management::permissions-by-product.adoc#runtime-manager[Access Management permission for Runtime Manager] is set to *Read Runtime Fabric*, which enables you to query Runtime Fabrics in your organization.
 
-=== Configure Your Local Registry and Install Runtime Fabric
+. In your shell script, run the following script, replacing `agent-version`, `authorization-token`, `mulesoft-docker-server`, and any other parameter value where appropriate:
 
-. Obtain `rtf-runtime-registry` credentials:
 +
 [source,copy]
 ----
-# rtfctl get registry-credentials <activation-data>
+
+##!/usr/bin/env bash
+dependencies=$(curl <platform-url>/runtimefabric/api/agentmanifests/<agent-version>         -H 'Authorization: bearer <authorization-token>' | jq -c '.dependencies |.[] | select(.provider | contains("generic")) | "\(.artifact):\(.version)"' | sed 's/"//g')
+for i in $dependencies
+do
+    echo "Processing $i"
+    docker pull <mulesoft-docker-server>/mulesoft/$i
+    docker tag  <mulesoft-docker-server>/mulesoft/$i <local-docker-server>/mulesoft/$i
+    docker push <local-docker-server>/mulesoft/$i
+done
 ----
 +
-This command sets `RTF_IMAGE_REGISTRY_ENDPOINT`, `RTF_IMAGE_REGISTRY_USER`, and `RTF_IMAGE_REGISTRY_PASSWORD` in the current shell environment. 
+[%header%autowidth.spread]
+.Script Parameters
+|===
+| Parameters | Description
+| `platform-url` | Anypoint Platform URL, such as https://anypoint.mulesoft.com, or https://eu1.anypoint.mulesoft.com
+| `agent-version` | Runtime Fabric agent release version required for the synchronization.
+| `authorization-token` |  Obtain the authorization token required for the synchronization through the connected apps capability and a link redirects to the connected apps page.
```

Comment:
```suggestion
| `authorization-token` |  Authorization token required for the synchronization through the connected apps capability and a link redirects to the connected apps page.
```

Reply:
I am confused by this wording. Is it the token that is providing the link to the connected apps page? Maybe:
"Authorization token required for the synchronization through the connected apps capability. When you use this token, a link redirects to the connected apps page.